### PR TITLE
[TCL30-67] Update Location card style

### PR DESCRIPTION
**Pull request by alxmcr**
## Description

As a user
I want to see a Location card with brand styles
so that I can identify the same style in our app

## Related Issue

[TCL30-67](https://the-collab-lab.atlassian.net/browse/TCL30-67)

## Acceptance Criteria

- Same styles in all location cards.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before

image

### After

image

## Testing Steps / QA Criteria

1. Checkout `ac-update-styles-location-card` branch.
2. Go to `/list`

### CODE REVIEW

**Comment by laurenmbeatty** [comment]
This is giving me low-contrast warnings. What about making it a button like the icon buttons instead? Otherwise I think you need to adjust either the font-size or color or both.
https://cdn.zappy.app/4c38e05cead7f0a89ffa82997822572e.png

**Comment by jssckbl** [comment]
I agree with what Lauren is suggesting, about the whole thing to be a functioning button. Then we can follow the design and keep all of the text throughout the card at a high contrast, and bopping in the accent color on the location pin image next to the distance so we can keep with a theme but not have low contrast with a very essential item.

**Comment by jssckbl** [comment]
Also it takes a lot to get everything on those cards! This is progressing nicely overall 🥳

**Comment by alxmcr** [comment]
> This is giving me low-contrast warnings. What about making it a button like the icon buttons instead? Otherwise I think you need to adjust either the font-size or color or both.
> https://cdn.zappy.app/4c38e05cead7f0a89ffa82997822572e.png

@laurenmbeatty & @jssckbl: I just added some changes. I hope it's much better.  Thanks!

image

**Comment by laurenmbeatty** [comment]
Better contrast...some of the images are overflowing the container image

**Comment by alxmcr** [comment]
> Better contrast...some of the images are overflowing the container image

Thanks! I just fixed this issue 😄 . Can you please check it again?

File: src/App.css [hunk 1]
```diff
@@ -696,6 +696,7 @@ nav {
   display: block;
   border-radius: 1rem;
   object-fit: cover;
+  height: var(--height-image-location-medium);
```
Comment (laurenmbeatty) [inline src/App.css:699]: 🎉

**Review by laurenmbeatty — APPROVED** [review]
LGTM 👍

**Pull request merged**